pprof/pprof.go

```diff
@@ -43,7 +43,7 @@ func Register(registrar tchannel.Registrar) {
 		req, err := thttp.ReadRequest(call)
 		if err != nil {
 			registrar.Logger().WithFields(
-				tchannel.LogField{"err", err.Error()},
+				tchannel.LogField{Key: "err", Value: err.Error()},
```

Comment:
+1, might need to change the API to just do `.With` where it takes key/value strings or something.